docs/rm/snax_cluster.md

```diff
@@ -0,0 +1,198 @@
+# SNAX Cluster Documentation
+
+This section discusses the modifications to the Snitch cluster to makthe acceleration extensions as easy as possible. The figure below shows a simplified figure of the SNAX architecture.
```

Comment:
Maybe tell the name of acceleration extensions, SNAX?
`make the`

Reply:
@xiaoling-yi  what is make the? hahahahaha

Reply:
but I added the meaning already thanks!